tests/e2e/functional_test.go

```diff
@@ -135,10 +135,16 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 		document("Verifying that the NumaflowControllerRollout was created")
 		Eventually(func() error {
-			_, err := numaflowControllerRolloutClient.Get(ctx, numaflowControllerRolloutName, metav1.GetOptions{})
+			_, err = numaflowControllerRolloutClient.Get(ctx, numaflowControllerRolloutName, metav1.GetOptions{})
 			return err
 		}, testTimeout, testPollingInterval).Should(Succeed())
 
+		document("Verifying that the NumaflowControllerRollout conditions are as expected")
+		Eventually(func() metav1.ConditionStatus {
+			rollout, _ := numaflowControllerRolloutClient.Get(ctx, numaflowControllerRolloutName, metav1.GetOptions{})
+			return getRolloutCondition(rollout.Status.Conditions, apiv1.ConditionChildResourceHealthy)
+		}).WithTimeout(testTimeout).Should(Equal(metav1.ConditionTrue))
```

Comment:
debating whether to bother you with this or not: I just updated all of these `Eventually()` calls to include a `testPollingInterval` (1 sec) instead of using the default 10 ms.  I wasn't sure if we could potentially stress the kubernetes client rate limiter by hitting it every 10ms or not.

Reply:
Can make this change really quick, no worries there.